### PR TITLE
Updated nmt_with_attention notebook links

### PR DESCRIPTION
**Pull request by JoKoum**

### CODE REVIEW

**Comment by JoKoum** [comment]
@googlebot I signed it!

File: site/en/tutorials/text/image_captioning.ipynb [hunk 1]
```diff
@@ -581,7 +581,7 @@
       "source": [
         "## Model\n",
         "\n",
-        "Fun fact: the decoder below is identical to the one in the example for [Neural Machine Translation with Attention](../sequences/nmt_with_attention.ipynb).\n",
+        "Fun fact: the decoder below is identical to the one in the example for [Neural Machine Translation with Attention](../../../../../../../text/blob/master/docs/tutorials/nmt_with_attention.ipynb).\n",
```
Comment (8bitmp3) [inline site/en/tutorials/text/image_captioning.ipynb:584]:
```suggestion
        "Fun fact: the decoder below is identical to the one in the example for [Neural Machine Translation with Attention](www.tensorflow.org/text/tutorials/nmt_with_attention).\n",
```

File: site/en/tutorials/text/image_captioning.ipynb [hunk 2]
```diff
@@ -1035,7 +1035,7 @@
       "source": [
         "# Next steps\n",
         "\n",
-        "Congrats! You've just trained an image captioning model with attention. Next, take a look at this example [Neural Machine Translation with Attention](../sequences/nmt_with_attention.ipynb). It uses a similar architecture to translate between Spanish and English sentences. You can also experiment with training the code in this notebook on a different dataset."
+        "Congrats! You've just trained an image captioning model with attention. Next, take a look at this example [Neural Machine Translation with Attention](../../../../../../../text/blob/master/docs/tutorials/nmt_with_attention.ipynb). It uses a similar architecture to translate between Spanish and English sentences. You can also experiment with training the code in this notebook on a different dataset."
```
Comment (8bitmp3) [inline site/en/tutorials/text/image_captioning.ipynb:1038]:
```suggestion
        "Congrats! You've just trained an image captioning model with attention. Next, take a look at this example [Neural Machine Translation with Attention]((www.tensorflow.org/text/tutorials/nmt_with_attention). It uses a similar architecture to translate between Spanish and English sentences. You can also experiment with training the code in this notebook on a different dataset."
```

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
Nice catch @JoKoum 

Let's use the full URL here (www.tensorflow.org/text/tutorials/nmt_with_attention).

Many thanks!!

**Review by 8bitmp3 — APPROVED** [review]

**Review by MarkDaoust — APPROVED** [review]

**Comment by 8bitmp3** [comment]
Good call @MarkDaoust Thanks

**Pull request merged**